Signal/src/ViewControllers/ConversationView/ConversationViewItem.m

```diff
@@ -304,7 +304,7 @@ - (DisplayableText *)displayableTextForInteractionId:(NSString *)interactionId
         NSString *text = textBlock();
 
         // Only show up to N characters of text.
-        const NSUInteger kMaxTextDisplayLength = 1024;
+        const NSUInteger kMaxTextDisplayLength = 512;
```

Comment:
Okay, for simplicity I've just dialed down the max character count.  I'm not sure exactly what the right value is.  Let's see if we feel like we're seeing "tap for more" too often.